api/serializers/personal_info.py

```diff
@@ -45,3 +45,17 @@ class Meta:
         model = PersonalInfo
         fields = ("registration_telegram_bot_chat_id",)
         extra_kwargs = {"registration_telegram_bot_chat_id": {"required": True}}
+
+
+class PublicPersonalInfoSerializer(serializers.ModelSerializer[PersonalInfo]):
+    """A serializer used to return public information about a user in Tooljet views."""
+
+    class Meta:
+        model = PersonalInfo
+        fields = (
+            "email",
+            "chatwoot_conversation_id",
+            "phone",
+            "registration_telegram_bot_chat_id",
+            "telegram_username",
+        )
```

Comment:
```suggestion
            "telegram_username",
            "utc_timedelta",
        )
```

Reply:
It already exists in CommonPublicXXXSerializer (for student and teacher). Do you suggest to move it to this personal info serializer? Than we won't see it in public views which are without personal info.

Reply:
Ah, okay.